### DISCUSSION TITLE
installation on two hard disk

### DISCUSSION THREAD

**Discussion by robertocannito** [SpiralLinux/SpiralLinux-project#122] [General]
I would like install Spiral on a notebook with 2 separate hd
on first one I would like install 
/boot/efi
/

On second 
/home

How do you suggest for the installation, considering that Calamares it's just a little bit freaken on this matter?
I would like avoid also problem with snapper

Thank since now

**Comment by geckolinux** [comment]
Hi there, thanks for the question. What you describe seems to work with Calamares, I just tried it in VirtualBox with the Manual Partitioning mode:

![Screenshot from 2022-09-14 08-34-44](https://user-images.githubusercontent.com/2692138/190168811-5b6dfc54-1cbc-49ec-bc41-bc86f67d3552.png)

You might experience a crash if one or both of the disks are completely new with no partition table, so maybe try pre-creating the partition table first with Gparted.

Please let us know how it worked for you! Thanks.

**Comment by robertocannito** [comment]
ok thank you all is ok
my only doubt is just for snapper, because in this case /home is installer ad partition and as volume, isn't?

**Reply by geckolinux** [reply]
Hi @robertocannito , yes, that's correct. It shouldn't cause any problems, but if it does please post here. Thanks!
